locale/en_US/api.po

```diff
@@ -47,6 +53,15 @@ msgstr "You do not have permission to delete this journal."
 msgid "api.contexts.404.contextNotFound"
 msgstr "The journal you requested was not found."
 
+msgid "api.dois.406.noActionIncluded"
+msgstr "No valid action was included with the request."
+
+msgid "api.dois.406.noRegistrationAgencyConfigured"
+msgstr "No DOI registration agency has been configured for this journal."
+
+msgid "api.dois.406.noSubmissionsIncluded"
+msgstr "No valid action was included with the request."
```

Comment:
I did purposefully use it, but I wasn't sure about it. Looking at it again, I misunderstood it (something about not acceptable header values rather than just values. What response code have you used for things like this?